lexers/h/hy_test.go

```diff
@@ -0,0 +1,20 @@
+package h_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/alecthomas/chroma"
+	"github.com/alecthomas/chroma/lexers/h"
+)
+
+func TestHy_AnalyseText(t *testing.T) {
+	data, err := ioutil.ReadFile("testdata/hy_basic.hy")
```

Comment:
Let's have 2 separate test cases, to test both variations individually.

Reply:
make sense tks.